packages/logger/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.0.0] - 2022-11-29
+
+### Added
+
+- Initial release
```

Comment:
Usually when its an initial release, I still like us to just explain a baseline of what is being launched. This helps to act as a baseline for future changes